packages/wdio-cucumber-framework/src/index.js

```diff
@@ -21,6 +21,14 @@ class CucumberAdapter {
         this.capabilities = capabilities
         this.config = config
         this.cucumberOpts = Object.assign(DEFAULT_OPTS, config.cucumberOpts)
+
+        /**
+         * gets current step data: `{ uri, feature, scenario, step, sourceLocation }`
+         * or `null` for some hooks.
+         *
+         * @return  {object|null}
+         */
+        this.getCurrentStep = () => this._getCurrentStep()
```

Comment:
The function is not overwritten anywhere.

`this.getCurrentStep` have to be defined at the beginning because it is used in `this.wrapSteps` function.
`this.wrapSteps` is called before `this._getCurrentStep` (please notice `_`) is defined.

Do you have any suggestions on how can it be improved? I can use a better function names at least

Reply:
Hold on, I think I now how it can be improved

Reply:
@christian-bromann removed it, please take another look